README.md

```diff
@@ -200,6 +201,35 @@ $ git fetch
 $ git checkout v0.3.0
 ~~~
 
+If you've [installed via Homebrew](#homebrew-on-mac-os-x), then upgrade
+via its `brew` command:
+
+~~~ sh
+$ brew update
+$ brew upgrade jawshooah/nodenv/nodenv node-build
```

Comment:
Does upgrade work correctly in this situation? It has been awhile since I was on Mac, but IIRC I had to `brew reinstall --HEAD ...` for it to upgrade correctly.